src/browser/api/external_application.ts

```diff
@@ -10,7 +10,27 @@ import { Identity } from '../../shapes';
 import * as ProcessTracker from '../process_tracker.js';
 import route from '../../common/route';
 
-const authenticatedConnections: any[] = [];
+const authenticatedConnections: ExternalApplicationOptions[] = [];
+
+export interface ExternalApplicationClient {
+    type: string;
+    version: any;
+}
+
+export interface ExternalApplicationOptions extends Identity {
+    runtimeClient: boolean;
+    nonPersistent: boolean;
+    id: number;
+    licenseKey?: string;
+    configUrl?: string;
+    pid?: number;
+    client?: ExternalApplicationClient;
+    type?: string;
```

Comment:
As discussed perhaps an inline comment to explain the meaning of `type` here. Specifically, this is an authorization type.